lib/Mojolicious/Controller.pm

```diff
@@ -768,6 +773,13 @@ current request.
   # "/list?q=mojo&page=2" if current request was for "/list?q=mojo&page=1"
   $c->url_with->query({page => 2});
 
+
+=head2 url_for_asset
+
+  my $url = $c->url_for_asset('/app.js');
```

Comment:
I think it would make sense to just use "app.js" and not "/app.js" in the example here, since it's relative to the assets directory.